microfs.py

```diff
@@ -102,9 +108,7 @@ def execute(commands, serial=None):
             serial.write(command_bytes[i:min(i + 32, len(command_bytes))])
             time.sleep(0.01)
         serial.write(b'\x04')
-        response = bytearray()
-        while not response.endswith(b'\x04>'):  # Read until prompt.
-            response.extend(serial.read_all())
+        response = serial.read_until(b'\x04>')       # Read until prompt.
```

Comment:
This serial call was changed because `read_until` has a built-in time out (the PySerial default if not set in the `Serial` constructor), so if there are issues it will not stay in that while loop forever.